src/lifish/preferences_persistence.hpp

```diff
@@ -5,7 +5,12 @@
 
 namespace lif {
 
+#if defined(SFML_SYSTEM_MACOS)
+// Preferences file shouldn't be hidden
```

Comment:
Any reason why it shouldn't? 
Moreover, any reason why it shouldn't _specifically_ on mac?

Reply:
Users might be surprised in case they open the `~/Library/ApplicationSupport/BOOM Remake` folder and see that it's "empty". Honestly, it shouldn't be hidden on other platforms too. On Linux it should be saved in `XDG_DATA_HOME/BOOM Remake/...` or `XDG_CONFIG_HOME/BOOM Remake/...` and `AppData` on Windows.